scripts/diagnose.sh

```diff
@@ -142,6 +142,7 @@ commands=(
 	'$ENG logs resin_supervisor'
 	'curl --max-time 5 localhost:'"${LISTEN_PORT}"'/v1/healthy'
 	'journalctl --no-pager --no-hostname -n 200 -a -u resin-supervisor'
+	'ls -R /tmp/*-supervisor/**/*'
```

Comment:
Would it be worth adding `-l` in order to get the timestamp?  Other than that, :+1: